wiki/CreatingMissions.md

```diff
@@ -328,10 +326,22 @@ This specifies that the mission does not show up in the player's list of mission
 
 If a mission is marked with `priority`, only other "priority" missions can be offered alongside it.
 
-If a mission is marked with `minor`, it will be offered only if no other missions are being offered at the same time.  
+If a mission is marked with `minor`, it will be offered only if no other missions are being offered at the same time, including other minor missions. See the "order" attribute below to learn how to influence precedence if required.
```

Comment:
```suggestion
If a mission is marked with `minor`, it will be offered only if no other missions are being offered at the same time, including other `minor` missions. See the `order` attribute below to learn how to influence precedence if required.
```